trafilatura/utils.py

```diff
@@ -454,3 +454,14 @@ def text_chars_test(string: Optional[str]) -> bool:
     # or not re.search(r'\w', string)
     # return string is not None and len(string) != 0 and not string.isspace()
     return bool(string) and not string.isspace()  # type: ignore[union-attr]
+
+
+def copy_attributes(dest_elem: _Element, src_elem: _Element) -> None:
+    '''Copy attributes from src element to dest element'''
+    for key in src_elem.keys():
+        dest_elem.set(key, src_elem.attrib[key])
+
+
+def is_in_table_cell(elem: _Element) -> bool:
+    '''Check whether an element is in a table cell'''
+    return elem.xpath('//ancestor::cell')
```

Comment:
@unsleepy22 The `.xpath` method returns a list of elements or `[]`, the type annotation is not correct and since we don't need the list it's useless to return it, I suggest using `return bool(elem.xpath(...))`.

Reply:
Yes you're right, type check didn't find this one.

Reply:
Fixed, could you take a look again?